CMakeLists.txt

```diff
@@ -26,10 +26,11 @@ include(AddWarningsConfigurationToTargets)
 
 include(CMakePackageConfigHelpers)
 
+set(UnicyclePlanner_VERSION 0.1.100)
 
 project(UnicyclePlanner
         LANGUAGES CXX
-        VERSION 0.1.100)
+        VERSION ${UnicyclePlanner_VERSION})
```

Comment:
This variable is already defined by the call to `project`, no need to set it manually, see https://cmake.org/cmake/help/v3.10/variable/PROJECT-NAME_VERSION.html#variable:%3CPROJECT-NAME%3E_VERSION .